### ISSUE TITLE
High vulnerability in dependency package vinyl-fs > glob-stream > glob-parent

### ISSUE THREAD

**Issue by pedrodlz** [i18next/i18next-parser#684]
## 🐛 Bug Report

High vulnerability found in package glob-parent
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ high                               │ glob-parent before 5.1.2 vulnerable to Regular Expression                                                                   │
│                                       │ Denial of Service in enclosure regex                                                                                                       │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package                         │ glob-parent                                                                                                                                            │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in                      │ >=5.1.2                                                                                                                                                   │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of              │ i18next-parser                                                                                                                                         │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path                               │ i18next-parser > vinyl-fs > glob-stream > glob-parent                                                                         │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info                       │ https://www.npmjs.com/advisories/1081884                                                                                        │
└───────────────┴──────────────────────────────────────────────────────────────┘

## To Reproduce

```js
yarn audit
```

## Expected behavior

No vulnerabilities found

## Your Environment

- *runtime version*: node v16.16.0, Ract Native 0.68.5
- *i18next version*: 6.6.0
- *os*: Windows

**Comment by eelkevdbos** [comment]
If you run npm v8.3, you can add an override to package.json until it is fixed:

```json
  "overrides": {
    "glob-parent": "^5.1.2"
  }
```

**Comment by chidg** [comment]
Hi maintainers, any chance of an update on this? `overrides/resolutions` is an ok workaround, but it would be nicer if there was a proper fix.

**Comment by karellm** [comment]
@chidg I have limited bandwidth atm, would you be available to make a PR?

**Comment by chidg** [comment]
Hi @karellm, it turns out you already have a PR for the dependency upgrade: #859 
Unfortunately there is a failing test relating to Broccoli. I spent some time debugging it but couldn't make much progress.

**Comment by karellm** [comment]
I've spent about an hour trying to debug this and couldn't find why the test fail. It looks like the files are not being parsed for some obscure reason.

**Comment by chidg** [comment]
Same experience I had!

**Comment by woutervanvliet** [comment]
Spent well over an hour debugging; turned out glob-stream, used by vinyl-fs, only scans from the current directory and upwards, thus not finding temporary files created by Broccoli and in turn not creating translation files.

This all should be solved in #930.

**Comment by karellm** [comment]
This should be fixed as of `8.9.0`
